Quaternion/tests/test_all.py

```diff
@@ -30,6 +169,59 @@ def test_from_transform():
     assert np.allclose(q.roll0, 30)
     assert np.allclose(q.ra0, 10)
 
+    t = [[9.25416578e-01,  -3.18795778e-01,  -2.04874129e-01],
+         [1.63175911e-01,   8.23172945e-01,  -5.43838142e-01],
+         [3.42020143e-01,   4.69846310e-01,   8.13797681e-01]]
+    _ = Quat(t)
+
+
+def test_from_transform_vectorized():
+    q = Quat(transform=transform_23)
+    assert q.q.shape == (2, 3, 4)
+    for i in indices(equatorial_23.shape[:-1]):
+        # check that Quat(transform).q[i] == Quat(transform[i]).q
+        assert np.all(q.q[i] == Quat(transform=transform_23[i]).q)
+
+    q = Quat(transform=transform_23[:1, :1])
+    assert q.q.shape == (1, 1, 4)
+
+    t = [[[[9.25416578e-01, -3.18795778e-01, -2.04874129e-01],
+           [1.63175911e-01, 8.23172945e-01, -5.43838142e-01],
+           [3.42020143e-01, 4.69846310e-01, 8.13797681e-01]]]]
+    q = Quat(transform=t)
+    assert q.q.shape == (1, 1, 4)
+
+
+def test_eq_from_transform():
+    # this raises 'Unexpected negative norm' exception due to roundoff in copy/paste above
+    # q = Quat(transform=transform_23)
+    # assert q.equatorial.shape == (2, 3, 3)
+    # assert np.allclose(q.equatorial, equatorial_23)
+
+    # this one fails (quaternion -> equatorial -> quaternion is not an identity)
```

Comment:
This is now tracked with #10 and you can remove these comments and not worry about it for this PR.

Reply:
removed